tools/build/configure.py

```diff
@@ -887,7 +894,15 @@ def build(
                         )
                     elif name.endswith("_tex"):
                         compress = False
-                        bin_path = path
+                        tex_dir = path.parent / name
+                        build(
+                            bin_path,
+                            [tex_dir],
+                            "tex",
+                            variables={
+                                "tex_dir": str(tex_dir)
```

Comment:
Is `str()` required? Elsewhere it doesn't seem to be

Reply:
maybe you're thinking of something else? I see lots of calls to str() to convert Paths to strings here, since the variables dict is typed as str,str